exasol_script_languages_container_ci/lib/release_upload.py

```diff
@@ -24,6 +24,16 @@ def _parse_repo_url(source_repo_url: str) -> str:
     return res.groups()[0]
 
 
+def _upload_assets(release_artifacts: Iterable[str], repo_id: str, release_id: int, content_type: str,
+                   release_uploader: GithubReleaseAssetUploader, label_prefix: str):
+
+    for release_artifact in release_artifacts:
+        label = Path(release_artifact).name.split(".")[0]
+        release_uploader.upload(archive_path=release_artifact,
```

Comment:
What is the effect of the label used here. Are the file downloaded from the release still have their original names?

Reply:
I use the label to remove the path (`.name`)

Reply:
The idea was to get the prefix of the flavor, like 
`flavor-abc.tar.gz` => `flavor-abc`
`flavor-abc.tar.gz.shasum512` => `flavor-abc`

But now I realized that we also used dots in the flavor names itself :( So this does not work. I simply will use the whole file name as label now